articles/Object-Detection-using-Fast-R-CNN-BrainScript.md

```diff
@@ -0,0 +1,537 @@
+---
+title:   Object  detection using Fast R-CNN
+author:    pkranen
+ms.author:   pkranen
+ms.date:   06/27/2017
+ms.custom:   cognitive-toolkit
+ms.topic:   conceptual
+ms.service:  Cognitive-services
+ms.devlang:   brainscript
+---
+
+# Object  detection using Fast R-CNN
+
+**Table of Contents**
+
+- [Summary](#summary)
+- [Setup](#setup)
+- [Run the toy example](#run-the-toy-example)
+- [Run Pascal VOC](#run-pascal-voc)
+- [Train CNTK Fast R-CNN on your own data](#train-on-your-own-data)
+- [Technical details](#technical-details)
+- [Algorithm details](#algorithm-details)
+
+## Summary
+
+<p align="center">
+<a target="_blank" href="./Tutorial_FastRCNN/svm_4WIN_20160803_12_37_07_Pro.jpg"><img src="./Tutorial_FastRCNN/svm_4WIN_20160803_12_37_07_Pro.jpg" alt="image" height="250"/></a>
+<a target="_blank" href="./Tutorial_FastRCNN/bus_01.jpg"><img src="./Tutorial_FastRCNN/bus_01.jpg" alt="image" height="250"/></a>
+</p>
+
+**This tutorial describes how to use CNTK Fast R-CNN with BrainScript and cntk.exe. Fast R-CNN using the CNTK Python API is described [here](./Object-Detection-using-Fast-R-CNN.md).**
+
+The above are examples images and object annotations for the grocery data set (left) and the Pascal VOC data set (right) used in this tutorial.
+
+_Fast R-CNN_ is an object detection algorithm proposed by _Ross Girshick_ in 2015.
+The paper is accepted to ICCV 2015, and archived at https://arxiv.org/abs/1504.08083.
+Fast R-CNN builds on previous work to efficiently classify object proposals using deep convolutional networks.
+Compared to previous work, Fast R-CNN employs a _region of interest pooling_ scheme that allows to reuse the computations from the convolutional layers.
+
+Additional material: a detailed tutorial for object detection using CNTK Fast R-CNN with BrainScript 
+(including optional SVM training and publishing the trained model as a Rest API) can be found [here](https://github.com/Azure/ObjectDetectionUsingCntk).
+
+## Setup
+
+To run the code in this example, you need a CNTK Python environment (see [here](./Setup-CNTK-on-your-machine.md) for setup help). You need to work from your Python 3.4 environment (if you are using Anaconda Python type `activate cntk-py34` (from a standard command line, not PowerShell), assuming `cntk-py34` is your environment name). Further you need to install a few additional packages. From your Python 3.4 environment (64bit version assumed, Python 3.5 analogously), go to the FastRCNN folder and run:
+
+```
+pip install -r requirements.txt
+```
+
+Known issue: to install scikit-learn you might have to run `conda install scikit-learn` if you use Anaconda Python.
```

Comment:
>Known issue: [](start = 0, length = 12)

i'm confused how to set this up... 

is 'requirement.txt now sufficient or do I need all this?
numpy and scipy are cntk pre-req ... why would I need them. these are in my environment already ...